### DISCUSSION TITLE
Is it possible to deploy ViT using hls4ml?

### DISCUSSION THREAD

**Discussion by zjnyly** [fastmachinelearning/hls4ml#803] [Q&A]
Recently I am required to deploy a transformer model on FPGA using hls4ml by my mentor. 

I am not sure if this is possible, because the model is too large and may need to buffer a large amont of intermediate data.

So, is it possible, or worth to try?

**Comment by ayusdixit** [comment]
same issue with me

**Comment by liuzrcc** [comment]
Same
